### PR TITLE
get started update

### PR DESCRIPTION
**Pull request by sarahxxu**

### CODE REVIEW

**Comment by sarahxxu** [comment]
hi @duynguyen @meryllblanchet @icaraps this is the get started update I was referring to. Please review and merge. Thank you very much!

**Comment by meryllblanchet** [comment]
Looking at it now :)

File: getting_started/first_app.md [hunk 1]
```diff
@@ -1,125 +1,395 @@
 # Creating your First Project Firefly App
 
-## Creating a new Project on Developer Console
+In this tutorial, we'll guide you through the following steps to give you an introduction on how to set up and to developer a Project Firefly Application.
```
Comment (icaraps) [inline getting_started/first_app.md:3]: typo "to developer"

File: getting_started/first_app.md [hunk 2]
```diff
@@ -1,125 +1,395 @@
...
+1. You should see a new project generated with 2 default `Workspaces`. 
+    - Workspaces can be used to manage different deployment environments (dev, qa, stage, prod) for your application and to provide individual working environment for each developer on the project. Workspace is where you will connect services and get the credential details needed to connect to Adobe APIs. Connected services can differe from workspace to workspace, and the credentials used within each workspace is not shared across workspaces. 
+    - Each Project Firefly project has two default workspaces: `Production` and `Stage`. You can add more workspaces as needed. The `Production` workspace is specicial, as it is used for the submission and distribution flow. When you’re ready to deploy your app, you will submit it for approval from the Production workspace.
```
Comment (icaraps) [inline getting_started/first_app.md:44]: typo "specicial"

File: getting_started/first_app.md [hunk 3]
```diff
@@ -1,125 +1,395 @@
...
+    $ cd helloworld
+    $ ls
+    README.md		console.json		manifest.yml		package-lock.json	test
+    actions			e2e			node_modules		package.json		web-src
+    ```
+    
+### 4.2 Developer is not Logged in
+
+#### Developer with a Console config file
+This flow is intended for developers who do not have access to Adobe Developer Console, likely due to permission issues, but can get credentials that's tied to a Console workspace. For this flow to work, the developer should obtain ask somone with access to set up a project and a workspace following the last few sections. With the workspace correctly set up, the credentails can downloaded by authorized Console users through the `Download all` button in Workspace overview.
```
Comment (icaraps) [inline getting_started/first_app.md]: typo "should obtain ask somone" and "the credentails can downloaded"

File: getting_started/first_app.md [hunk 4]
```diff
@@ -1,125 +1,395 @@
...
-- `test`: including unit tests and integration tests
-- `e2e`: including end-to-end tests
-- `manifest.yml`: definition of I/O Runtime actions being deployed with the app
-- `package.json`: project definition and metadata
-- `.env`: containing environment variables that are useful for the app, e.g. I/O Runtime credentials and Adobe Product API tenant specifics (API key, secrets, etc.)
-- `.aio`: containing config variables that are useful for the AIO CLI to facilitate the app, e.g. supported API services
+Now that your project is initialized, let's open the project in your favorite IDE. We recommend using VSCode, you can easily open the project by entering `code <app-name>`, or opening VSCode -> Open... -> select app folder.
 
-## Local Development
+You should be able to see the these folders and files in your project:
```
Comment (icaraps) [inline getting_started/first_app.md:212]: typo "the these folders"

File: getting_started/first_app.md [hunk 5]
```diff
@@ -1,125 +1,395 @@
...
 
-### Running the Application
+1. `actions`: this folder is intended for backend source code for all serverless actions
+1. `web-src`: this folder is intended for frontend source code such as html templates, react components, JS, CSS
+1. `test`: this folder is intended for unit tests and integration tests
+1. `e2e`: this folder is intended for  end-to-end tests
+1. `manifest.yml`: this file describes the backend actions you would like to deploy or to redeploy. 
+    - The manifest file contents shoud adhere to the [OpenWhisk deployment YAML specification](https://github.com/apache/openwhisk-wskdeploy/tree/master/specification#package-specification). Once defined, the CLI use this file to deploy or redeploy actions. You might see values like `$CUSTOMER_PROFILE_TENANT` listed on this page. These are environment variables that you can define in your `.env` file. 
+1. `package.json`: this file describles project definition and various metadata relevant to the project.
```
Comment (icaraps) [inline getting_started/first_app.md:220]: typo "describles"

File: getting_started/first_app.md [hunk 6]
```diff
@@ -1,125 +1,395 @@
...
+    - It is used to give information to npm that allows it to identify the project as well as handle the project's dependencies. Learn more [here](https://nodejs.org/en/knowledge/getting-started/npm/what-is-the-file-package-json/).
+1. `.aio`: this file contains config variables that are useful for the AIO CLI to facilitate the app, e.g. supported API services. 
+    - You can manually update the file or use the `aio config` commands to add or to remove configurations. Learn more about the [Config Plugin](https://github.com/adobe/aio-cli-plugin-config). 
+1. `.env`: this file contains environment variables that are useful for the app during development, e.g. I/O Runtime credentials and Adobe Product API tenant specifics (API key, secrets, etc.)
+    - The environment variables defined here can be used in the application (e.g. in `manifest.yml`). If you've set up credentials for the selected workspaces, you should be able to see some of those values prepopulated upon initialization, like `AIO_runtime_auth` and `AIO_runtime_namespace`. 
+    - This file is automatically included in `.gitignore`. It is not intented be shared given the credentails and secrets listed.
```
Comment (icaraps) [inline getting_started/first_app.md:226]: typoe "credentails"

File: getting_started/first_app.md [hunk 7]
```diff
@@ -1,125 +1,395 @@
...
+We integrated our CLI with an open source project [wskdebug](https://github.com/apache/openwhisk-wskdebug) to support our debug functionalities. This tool allows you to develop and debug OpenWhisk actions in your favorite IDE or debugger with a fast feedback loop. It features:
+- full debugging of actions of the respective language runtime
+- automatic code reloading
+- LiveReload for web actions
+- auto-invoking of actions on code changes
+- or running any shell command such as a curl request on code changes
+
+To debug your application, you will need to install some additional dependencies if you only set up the required tools, see  `Optional tools` section in  [Setting up Your Environment](getting_started/setup.md). When you have the dependencies installed, make sure that local dev environment is running with `aio app run`, the `--local` flag is optional. 
+
+With your applicaiton running, you can easily:
```
Comment (icaraps) [inline getting_started/first_app.md:325]: typo "applicaiton"

File: getting_started/first_app.md [hunk 8]
```diff
@@ -1,125 +1,395 @@
...
+...
+
+To view your deployed application:
+  -> https://<namespace>.adobeio-static.net/<packagename>/index.html
+To view your deployed application in the Experience Cloud shell:
+  -> https://experience.adobe.com/?devMode=true#/custom-apps/?localDevUrl=https://<namespace>.adobeio-static.net/<packagename>/index.html
+Well done, your app is now online 🏄
+```
+
+Note the last section of the output `To view your deployed application`. There are 2 urls of the app shown by default, which allow access either to the CDN host or Adobe Experience Cloud. The URL format of the app on Experence Cloud should follow `ttps://experience.adobe.com/?devMode=true#/custom-apps/?localDevUrl=<your-app-url>`.
```
Comment (icaraps) [inline getting_started/first_app.md:382]: typo "Experence Cloud" and "ttps"

**Comment by icaraps** [comment]
Super useful 👍

**Comment by meryllblanchet** [comment]
Great job @sarahxxu !

**Pull request merged**